SmartDeviceLink-iOS.podspec

```diff
@@ -24,8 +24,12 @@ ss.public_header_files = [
 'SmartDeviceLink/SDLProxyListener.h',
 'SmartDeviceLink/SDLSecurityType.h',
 'SmartDeviceLink/SDLStreamingMediaManager.h',
+'SmartDeviceLink/SDLTouch.h',
 'SmartDeviceLink/SDLTouchManager.h',
 'SmartDeviceLink/SDLTouchManagerDelegate.h',
+'SmartDeviceLink/SDLConsoleController.h',
+'SmartDeviceLink/SDLDebugTool.h',
+'SmartDeviceLink/SDLDebugToolConsole.h',
```

Comment:
I synced up the .podspec files with the `SmartDeviceLink.h` file. I also added two missing public files to the `SmartDeviceLink.h` file.